static/faq.html

```diff
@@ -1,116 +1,225 @@
 <!DOCTYPE html>
+<!--meta data-->
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
 <html lang="en">
 
 <head>
-    <meta charset="utf-8">
-
     <title>FAQ</title>
+    <link rel="icon" href="tabIcon.png">
 
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-
-    <style>
-        #main-body {
-            background-color: black;
-            color: palegreen;
-            width: 50%;
-            margin: 0 auto;
-            font-family: monospace;
-            font-size: 16px;
-            padding: 1%;
-        }
+    <!--and for some reason it doesn't want to work here at all...strange-->
+    <script type="module">
+        import * as THREE from "https://cdn.skypack.dev/three@0.132.0";
```

Comment:
Since the JS has been extracted to an external file, shouldn't this code be removed?